site/app/Resources/views/notification/notification_inbox.html.twig

```diff
@@ -0,0 +1,77 @@
+        </div>
+    </div>
+    <ul class="all-notifications">
+        {# If there are any notifications #}
+        {% if (mgilet_notification_count(app.user)) %}
+            {% for notification in notificationList %}
+                {% if (notification.seen) == true %}
+                    {# Render Seen Notification #}
+                    <li class="inbox-seen">
+                        <div class="row">
+                            <div class="col-sm-8 col-lg-8 col-xs-8 col-sm-8 inbox-message">
+                                {{ notification.notification.message }}
+                            </div>
+                            <button class="col-xs-4 col-md-4 col-sm-4 text-right notification-remove"
+                                    data-notification="{{ notification.notification.id }}"
+                                    data-notifiable="{{ notification.notifiableEntity.id }}">
+                                <i class="fa fa-trash markasseen-icon"></i>
+                            </button>
+                            <span class="notification-date text-right" data-livestamp="{{ notification.notification.date|date("c") }}">
+                            </span>
+                        </div>
+                    </li>
+                {% else %}
+                    {# Render Unseen Notification #}
+                    <li class="inbox-unseen">
+                        <div class="row">
+                            <div class="col-sm-8 col-lg-8 col-xs-8 col-sm-8 inbox-message">
+                                {{ notification.notification.message }}
+                            </div>
+                            <button class="col-xs-2 col-md-2 col-sm-2 notification-markasseen"
+                                    data-notification="{{ notification.notification.id }}"
+                                    data-notifiable="{{ notification.notifiableEntity.id }}">
+                                <i class="fa fa-eye markasseen-icon"></i>
+                            </button>
+                            <button class="col-xs-2 col-md-2 col-sm-2 text-right notification-remove"
+                                    data-notification="{{ notification.notification.id }}"
+                                    data-notifiable="{{ notification.notifiableEntity.id }}">
+                                <i class="fa fa-trash markasseen-icon"></i>
+                            </button>
+                            <span class="notification-date text-right" data-livestamp="{{ notification.notification.date|date("c") }}">
+                            </span>
+                        </div>
+                    </li>
+                {% endif %}
+            {% endfor %}
+        {% else %}
+            <li>No Notifications for you, yet!</li>
+        {% endif %}
+    </ul>
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+    {{ encore_entry_script_tags('notification') }}
```

Comment:
Only include the JS for this page here (notification_inbox.js). All other JS should be included in the JS file itself (as import).